### ISSUE TITLE
Version 1.6.6 of transxchange2GTFS does not handle zipped files

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SEBDYA/googletransitdatafeed#272]
```
What steps will reproduce the problem?
1. Create a .zip file with multiple TXC XML files in it
2. Try to convert these using version 1.6.6 of the converter 

What is the expected output? What do you see instead?
Expect a set of GTFS files to be generated based on all the XML files input. 
Instead, the following output is observed:

transxchange2GTFS 1.6.6
Please refer to LICENSE file for licensing information
301_20101027.xml
302_20101027.xml
303_20101027.xml
304_20101027.xml
305_20101027.xml
306_20101027.xml
307_20101027.xml
112_20101027.xml
300_20101027.xml
transxchange2GTFS write error:
duplicate entry: stop_times.txt 

The output directory ends up containing a corrupt gtfs.zip file that can't be 
opened. 

What version of the product are you using? On what operating system?
Version 1.6.6, Windows 2008 server. 

Please provide any additional information below.
Running the convereter with a .zip file that contains only 1 XML file seems to 
work fine. Similarly, running the converter with a single XML file as input 
works fine. 

```

Original issue reported on code.google.com by `heals...@gmail.com` on 8 Feb 2011 at 5:31

**Comment by GoogleCodeExporter** [comment]
Original comment by `Joachim....@gmail.com` on 8 Feb 2011 at 6:08
- Added labels: **Language-Java**, **Priority-High**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in TransXChange2GTFS 1.7.0
```

Original comment by `Joachim....@gmail.com` on 10 Feb 2011 at 6:16
- Changed state: **Fixed**
